nifi-commons/nifi-utils/src/main/java/org/apache/nifi/util/file/FileUtils.java

```diff
@@ -592,4 +592,41 @@ public static boolean isSame(final File file1, final File file2) throws IOExcept
         return digest.digest();
     }
 
+    /**
+     * Returns the capacity for a given path associated to a container name
+     * @param containerName container name
+     * @param path path
+     * @return total space
+     * @throws IOException in case there is no space
+     */
+    public static long getContainerCapacity(final String containerName, final Path path) throws IOException {
+        if (path == null) {
```

Comment:
Based on the other comments I've left, it may make sense to remove these checks from the utility method and instead perform the checks from where these methods are invoked. Since in some instances, we have a `Path` already and other instances we have a `File`. Not sure... just offering it as a possibility.